### PR TITLE
solving 'undefined' problem

### PR DESCRIPTION
**Pull request by yangga**
Check it out~

### CODE REVIEW

**Comment by rafaelklaessen** [comment]
I'd prefer the solution on the ssr-fix branch if it works.

**Comment by rafaelklaessen** [comment]
Nevermind, didn't see your comment on the issue! Will merge :smile:

File: index.js [hunk 1]
```diff
@@ -1,3 +1,12 @@
+if(typeof window === 'undefined'){
```
Comment (rafaelklaessen) [inline index.js]: Really small detail, could you please add a space after `if` and before `{`? Thanks! :smile:

Comment (rafaelklaessen) [reply index.js]: Goes for the webpackjsonpreact `if` too

**Comment by yangga** [comment]
@rafaelklaessen It's not working if the codes moved into src/index.js. I already tested.
Anyway, i added spaces as you wish.
Thanks.

**Comment by rafaelklaessen** [comment]
Thank you!

**Pull request merged**